From 36cc6ee78434b1fb9b47cbf87405f1e32282cd5c Mon Sep 17 00:00:00 2001
From: Ian Wahbe <ian@wahbe.com>
Date: Tue, 3 Oct 2023 12:19:15 -0700
Subject: [PATCH 1/2] Fix go module path: `terraform-providers` -> `Venafi`

---
 go.mod  | 4 ++--
 main.go | 5 +++--
 2 files changed, 5 insertions(+), 4 deletions(-)

diff --git a/go.mod b/go.mod
index 0324614..05168b8 100644
--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,6 @@
-module github.com/terraform-providers/terraform-provider-venafi
+module github.com/Venafi/terraform-provider-venafi
 
-go 1.12
+go 1.19
 
 require (
 	github.com/Venafi/vcert/v4 v4.22.0
diff --git a/main.go b/main.go
index 13942e8..7a7e9ad 100644
--- a/main.go
+++ b/main.go
@@ -2,9 +2,10 @@ package main
 
 import (
 	"flag"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/plugin"
-	"github.com/terraform-providers/terraform-provider-venafi/venafi"
 	"log"
+
+	"github.com/Venafi/terraform-provider-venafi/venafi"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/plugin"
 )
 
 func main() {
-- 
2.42.0

